elytron/src/test/resources/org/wildfly/extension/elytron/realms-test.xml

```diff
@@ -55,6 +55,11 @@
         <token-realm name="OAuth2Realm" principal-claim="sub">
             <oauth2-introspection client-id="a" client-secret="b" introspection-url="https://localhost/token/introspect" client-ssl-context="ClientCaSslContext" host-name-verification-policy="ANY" />
         </token-realm>
+
+        <jaas-realm name="myJaasRealm"  entry="Entry1" >
```

Comment:
All attributes allowing expressions should be tested with expressions set (somewhere). I'm not sure whether this is the right place but I haven't found another one.

Reply:
@OndrejKotek changed this to use expression.